addons/account/report/account_invoice_report.py

```diff
@@ -95,12 +95,12 @@ def _select(self):
                 move.invoice_date_due,
                 uom_template.id                                             AS product_uom_id,
                 template.categ_id                                           AS product_categ_id,
-                line.quantity / NULLIF(COALESCE(uom_line.factor, 1) / COALESCE(uom_template.factor, 1), 0.0) * (CASE WHEN move.move_type IN ('in_invoice','out_refund','in_receipt') THEN -1 ELSE 1 END)
+                line.quantity / NULLIF(COALESCE(uom_line.factor, 1) * COALESCE(uom_template.factor, 1), 0.0) * (CASE WHEN move.move_type IN ('in_invoice','out_refund','in_receipt') THEN -1 ELSE 1 END)
```

Comment:
This one is inside of a parenthesis so this change is wrong.

Reply:
Yes I thought the nullif is only for the uom_line.factor, but generally I think only the uom_line.factor should be inside the nullif as other parts should be multiplied